visualization/dashboard/src/Localization/en.json

```diff
@@ -335,7 +335,7 @@
         "cohort": "Cohort"
     },
     "DatasetExplorer": {
-        "helperText": "Learn about the over/underpresentation in your dataset. Create dataset cohorts above to analyze statistics along filters such as predicted outcome, dataset features and error groups. Use the x-axis and color selectors to further slice your dataset into more dimensions. Use the gear icon in the graph to change graph types.",
+        "helperText": "Explore your dataset statistics by selecting different filters along the X, Y, and color axis to slice your data into different dimensions. Create dataset cohorts above to analyze dataset statistics along filters such as predicted outcome, dataset features and error groups. Use the gear icon in the upper right-hand corner of the graph to change graph types.",
```

Comment:
nit: not sure it's possible to slice something *into* dimensions - maybe "along" instead?

Reply:
And then to avoid repetition the "along filters" in the next sentence should change to "using filters"

Reply:
@minthigpen I think I agree with Walter's suggestions, what do you think?